Sources/StreamChatUI/ChatChannelList/ChatChannelSwipeableListItemView.swift

```diff
@@ -59,7 +59,7 @@ open class _ChatChannelSwipeableListItemView<ExtraData: ExtraDataTypes>: _View,
         trailingConstraint?.isActive = true
         
         addSubview(bottomSeparatorView)
-        bottomSeparatorView.heightAnchor.pin(equalToConstant: 0.4).isActive = true
+        bottomSeparatorView.heightAnchor.pin(equalToConstant: 1 / contentScaleFactor).isActive = true
```

Comment:
What is `contentScaleFactor`? Would you mind adding documentation there?

Reply:
The documentation is already provided in the property itself, but I think it is a good idea to comment on what is that doing and why it is need, so I'll grab the commit's description and add it as a comment 👍

Reply:
Added the comment :)